Projects/Core/HGLogger/Sources/Logger.swift

```diff
@@ -6,3 +6,41 @@
 //
 
 import Foundation
+import os.log
+
+public final class Logger: Loggable {
+    private let subsystem: String
+    private let minLevel: LogLevelType
+
+    public init(
+        subsystem: String = Bundle.main.bundleIdentifier ?? "default",
+        minLevel: LogLevelType = .debug
+    ) {
+        self.subsystem = subsystem
+        self.minLevel = minLevel
+    }
+
+    public func log(
+        _ message: String,
```

Comment:
현재 로그가 String타입만 받고있긴한데
```suggestion
    _ message: @autoclosure () -> T
    
    let message = String(reflecting: message())
```
방식으로 오브젝트자체를 던질수도있긴해!! 
근데 `"\(object)"를 넣어도 똑같긴해서 
편한대로 고민해보면 좋을듯해서 의견남겨용

Reply:
> 인스턴스 

HGLogger().log("\(object)", level: .warning) // 수정전
HGLogger().log(object, level: .warning) // 수정후 

> 문자열

HGLogger().log("문자열", level: .warning) // 수정전
HGLogger().log("문자열", level: .warning) // 수정후

쓰는차이는 이렇게다를듯

Reply:
음 고민해보겠습네다~